wallet/wallet.go

```diff
@@ -3773,9 +3773,33 @@ func (w *Wallet) TotalReceivedForAddr(addr dcrutil.Address, minConf int32) (dcru
 	return amount, nil
 }
 
-// SendOutputs creates and sends payment transactions. It returns the
-// transaction hash upon success
+// CreateTxOptions allows changing the default behavior of how outputs are constructed
+// when calling `SendOutputsWithOptions`
+type CreateTxOptions struct {
+	// Determines whether the fee comes from the change output or from the payment recipient
+	RecipientPaysFee bool
+}
+
+// NewCreateTxOptions constructs an instance of `CreateTxOptions`, which allows
+// configuring how a transaction is built via methods that accept it as a parameter.
+func (w *Wallet) NewCreateTxOptions(recipientPaysFee bool) *CreateTxOptions {
+	return &CreateTxOptions{
+		RecipientPaysFee: recipientPaysFee,
+	}
+}
+
+// SendOutputs creates and sends payment transactions.
+// It returns the transaction hash upon success.
+//
+// When building the transaction, miner fees are subtracted from the change output.
```

Comment:
Also add a comment line saying:

`// Deprecated: Use SendOutputsWithOptions`

so that it comes up in the search of functions to remove or change when we do a major bump.